frontend/src/app/models/station.model.ts

```diff
@@ -1,8 +1,10 @@
+import { LatLngExpression } from "leaflet";
+
 export interface Station {
   _id?: string;
   cityId: string;
   name: string;
-  coordinates?: string;
+  coordinates?: LatLngExpression | undefined;
```

Comment:
zasto i upitnik undefined, msm da mozes obrisat jedno ako ista upitnik

Reply:
može u suštini bez ' | undefined' dijela, kad dodaš ? u interface prop onda ts automatski seta | undefined na tip koji si stavio

dodam ja u posebnom commitu nakon što ovo merge